modules/tinymce/CHANGELOG.md

```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config. #TINY-9653
+- New optional `defaultSelectedId` option to the `tree` component config. #TINY-9715
 
 ### Fixed
+- In the tree component, a selected item in a directory would not stay selected after collapsing the directory. #TINY-9715
```

Comment:
How can this be a bug, if it just has been implemented? It seems to me the entry in Added section is enough.

Reply:
Well there are two things that changed here. an internal change and an external (api) change. The internal change was the fix to the bug while the external change is an addition. Previously if you select a leaf inside a directory and then collapse that directory and expand it again, the item does not stay selected, hence the "bug" fix. And the added api `defaultSelectedId` is provided to the implementer to be able to keep the selected state of a leaf between redials.